src/driver.rs

```diff
@@ -299,6 +299,21 @@ pub fn main() {
         rustc_driver::catch_fatal_errors(move || {
             let mut orig_args: Vec<String> = env::args().collect();
 
+            if orig_args.get(1) == Some(&"rustc".into()) {
+                // first (0) arg is the bin path, second arg is "rustc"
+                // pass all succeeding args to rustc
+                let args_for_rustc = &orig_args[2..].to_vec();
+
+                let exitstatus = Command::new("rustc")
```

Comment:
No, this doesn't help. `rustc` isn't necessarily in the path, and even if it were, doesn't necessarily have anything to do with the version of rustc `clippy-driver` was built against. If I wanted to know the version of the `rustc` executable, I could just run `rustc --version`.